packages/core/src/components/dropdown/dropdown-button.scss

```diff
@@ -24,6 +24,12 @@
 
     &.value {
       color: var(--tds-dropdown-value-color);
+      font: var(--tds-detail-02);
+      letter-spacing: var(--tds-detail-02-ls);
+
+      /** Needed to not get be affected by overflow:hidden 
+      The original line-height is 1.14 */
+      line-height: 1.2;
```

Comment:
Setting 1.4 here locally works.

Reply:
Alright, looked good on my end. I bumped it to 1.3 now, please try it out again